lib/base-compiler.js

```diff
@@ -440,6 +474,39 @@ Compile.prototype.processAstOutput = function (output) {
     return output;
 };
 
+Compile.prototype.processRtlOutput = function (opts) {
+    var root_dir = path.dirname(opts.inputFilename);
+    var all_files = fs.readdirSync(root_dir);
+    var base = path.basename(opts.inputFilename);
+
+    var all_pass = [];
+
+    for (var i in all_files) {
+	var pass_str_idx = all_files[i].indexOf(base + '.');
+	if ( pass_str_idx != -1) {
+	    all_pass.push(all_files[i].substring(base.length + 1));
+	}
+    }
+    var output = {
+	all: all_pass,
```

Comment:
We usually remove the trailing comma from  this

Reply:
I like it that when you add a new line, the diff does not show the previously last line changed. I'll change this !